README.md

```diff
@@ -13,11 +13,18 @@
 [![Java Version](https://img.shields.io/badge/java-SE%2021-yellowgreen)](#download-and-installation)
 
 
-JPlag is a system that finds similarities among multiple sets of source code files. This way it can detect software plagiarism and collusion in software development. JPlag currently supports various programming languages, EMF metamodels, and natural language text.
+JPlag finds pairwise similarities among a set of multiple programs. It can reliably detect software plagiarism and collusion in software development, even when obfuscated. JPlag currently supports various programming and modeling languages. Moreover, it has rudimentary support for natural language.
```

Comment:
```suggestion
JPlag finds pairwise similarities among a set of multiple programs. It can reliably detect software plagiarism and collusion in software development, even when obfuscated. JPlag supports a large number of programming and modeling languages. Moreover, it has rudimentary support for natural language.
```